src/cryptography/hazmat/primitives/kdf/kbkdf.py

```diff
@@ -114,17 +135,27 @@ def derive(self, key_material: bytes, prf_output_size: int) -> bytes:
         if rounds > pow(2, len(r_bin) * 8) - 1:
             raise ValueError("There are too many iterations.")
 
+        fixed = self._generate_fixed_input()
+
+        if self._location == CounterLocation.BeforeFixed:
+            data_before_ctr = b""
+            data_after_ctr = fixed
+        elif self._location == CounterLocation.AfterFixed:
+            data_before_ctr = fixed
+            data_after_ctr = b""
+        else:
+            if self._break_location > len(fixed):
+                raise ValueError("break_location offset > len(fixed)")
```

Comment:
Do we need to update the documentation to reflect this ValueError exception?

Reply:
Nah.

Reply:
Being pedantic now, but, technically we could raise this `ValueError` from inside `__init__()`. We can calculate `fixed` there and check `break_location` at the end of `__init__()`.

Decided against it since it will introduce further changes and increase the scope of the MR...